internal/mode/static/state/change_processor_test.go

```diff
@@ -733,17 +722,6 @@ var _ = Describe("ChangeProcessor", func() {
 				},
 				)
 			})
-			When("the first Gateway update without generation changed is processed", func() {
```

Comment:
`change_processor.go` should not handle any cases with the same generation, as these should already be filtered out by the controller's `GenerationChangedPredicate`.